addons/common/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="Common">
         <Key ID="STR_CBA_Common_Component">
             <English>Community Base Addons - Common Component</English>
+            <German>Community Base Addons - Gemeinsame Komponente</German>
```

Comment:
auch: Allgemeine Komponente

Reply:
I always understood this to be the "shared component" - stuff that is used in multiple places. That is at least how we treated it in ACE: stuff that is used in at least two places gets moved to ACE Common, although I realize that not everything in CBA Common or ACE Common is shared between two other components or even used by a single other one.

I would've used "Allgemeine Komponente" if it were named "CBA General" instead of "CBA Common".

Reply:
Yea from this perspective you convinced me. Good job. Go into politics! 📦